In this patch, i changed the default name of the log file.
By default, the name is debug.log, i changed it to debug-$USER.log
where $USER is the user which runs the program.
--- TempFolder.java.orig	2010-03-16 03:40:25.000000000 +0100
+++ TempFolder.java	2010-03-16 01:47:43.000000000 +0100
@@ -1,8 +1,12 @@
+/*
+ * I Changed the default temp dir /tmp/javaps3media to
+ * /tmp/javaps3media-$(USER) 
+ */
+
 package net.pms.configuration;
 
 import java.io.File;
 import java.io.IOException;
-
 import net.pms.PMS;
 
 import org.apache.commons.io.FileUtils;
@@ -13,8 +17,12 @@
  * @author Tim Cox (mail@tcox.org)
  */
 class TempFolder {
-
-	private static final String DEFAULT_TEMP_FOLDER_NAME = "javaps3media";
+    
+    // Added by the debian package maintainer
+	String username = System.getProperty("user.name");
+	//private static final String DEFAULT_TEMP_FOLDER_NAME = "javaps3media-"+username;
+	private String DEFAULT_TEMP_FOLDER_NAME = "javaps3media-"+username;
+	// end modification
 
 	private final String userSpecifiedFolder;
 	
@@ -59,7 +67,8 @@
 		return folderFile;
 	}
 
-	private static File getSystemTempFolder() throws IOException {
+		//private static File getSystemTempFolder() throws IOException {
+		private File getSystemTempFolder() throws IOException {
 		File tmp = new File(System.getProperty("java.io.tmpdir")); //$NON-NLS-1$
 		File myTMP = new File(tmp, DEFAULT_TEMP_FOLDER_NAME); //$NON-NLS-1$
 		FileUtils.forceMkdir(myTMP);
